### ISSUE TITLE
[Suggestion]: When Immersive Portals is installed, bowels entrance becomes an immersive portal

### ISSUE THREAD

**Issue by Cod1ngGamer** [nonamecrackers2/crackers-wither-storm-mod#1912]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Basically, when Immersive Portals is installed, instead of the bowels entrance just being a purple hole, it's actually a portal into the bowels, like the immersive nether and end portals. Here's an example of a nether portal in Immersive Portals:
![image](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/137913869/93c54516-7487-42e4-a1d9-8c683418eb1c)
Basically the way it works is it lets you seamlessly transition between dimensions, and it's really cool. I think having a portal like that for the Bowels would be absolutely incredible. Plus, it could finally solve the dispute of whether the Bowels are actually sideways or not.

**Comment by WitherBean** [comment]
This was already suggested.

**Comment by nonamecrackers2** [comment]
#1016
